SU-SOE/stanford_soe_helper
# Soe 3071 check solr

# READY FOR REVIEW

# Summary
- Adding check for solr.

# Needed By (Date)
- Dynomite release

# Urgency
- Mild

# Steps to Test

1. Clone site to local
3. Check out this branch
2. Set solr index to read only
3. run cron
4. review dblog for Solr status

# Affected Projects or Products
- SOE

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-3071
